Counter/src/app/home/home.component.html

```diff
@@ -1 +1,7 @@
-<p>home works!</p>
+<div class="list">
+    <div class="row">
```

Comment:
Instead of classes for list and row, you should try and use a more semantic markdown approach and use `ul` and `li` elements. Then you won't need classes to indicate what the divs are. You can change the styling in the CSS for `ul` and `li` to do the same styling you have for the classes and remove the dots that get added to the `ul` with other css properties like `list-style`.

Reply:
I added row since it's a Bootstrap container and I wanted to have more than one thumbnail show up in a row. I do agree that the list div is a bit misleading and I can change the name.